src/main/java/edition/academy/seventh/serivce/PromotionProvider.java

```diff
@@ -12,11 +10,11 @@
  * @author Bartosz Kupajski
  *
  */
-public interface IPromotionScrapping {
+public interface PromotionProvider {
 
   /**
```

Comment:
Co robi ta metoda? Uaktualnić dokumentację. Czy mam to puścić bez aktualizacji i będzie na to kwit?